docs_user/modules/proc_adopting-image-service-with-object-storage-backend.adoc

```diff
@@ -1,8 +1,8 @@
 [id="adopting-image-service-with-object-storage-backend_{context}"]
 
-= Adopting the {image_service} that is deployed with a {object_storage} backend
+= Adopting the {image_service} that is deployed with a {object_storage} back end
 
-Adopt the {image_service_first_ref} that you deployed with an {object_storage_first_ref} backend. When {image_service} is deployed with {object_storage_first_ref} as a backend in the {rhos_prev_long} environment based on {OpenStackPreviousInstaller}, the control plane `glanceAPI` instance is deployed with the following configuration:
+Adopt the {image_service_first_ref} that you deployed with an {object_storage_first_ref} back end in the {rhos_prev_long} ({OpenStackShort}) environment. The control plane `glanceAPI` instance is deployed with the following configuration:
 
```

Comment:
Can we provide more context as to what this glanceAPI configuration will be used for? Something like, “You use this configuration in the patch manifest that deploys the image service with the object storage back end.”

Reply:
+1 to have more context and I'm ok to the sentence you proposed. I think it would be enough as the following snippet providing the glance config is clear enough.